generator.py

```diff
@@ -1,7 +1,9 @@
-#!/usr/bin/env python3
```

Comment:
Please revert this change. The path isn't always `/usr/bin` for python3 (in my case it's in `sbin` for example :p)